_schemaV2.graphql

```diff
@@ -1033,29 +1033,15 @@ type ArtistTargetSupplyMicrofunnel {
 
     # Randomize the order of artworks for display purposes.
     randomize: Boolean
-
-    # Number of artworks to return
-    size: Int
```

Comment:
Note that this `size` argument is removed from the new shared type, because I don't think it's needed now that we're using connections (which have a `first` argument).